rocket_test.go

```diff
@@ -0,0 +1,20 @@
+package rocket_test
+
+import (
+	"testing"
+
+	"github.com/dannypsnl/rocket"
+)
+
+func TestDuplicateRoutePanic(t *testing.T) {
```

Comment:
I think this test is hard to understand because you cannot saw the `/` been mounted twice in this test case.
but I think this problem is because our test is the mess now. we should open a new issue to deal with this.

Reply:
So what should I do for the test?

Reply:
nothing for now, I will open a new issue and give an example.